pkg/core/bootstrap/bootstrap.go

```diff
@@ -4,6 +4,8 @@ import (
 	"context"
 	"net"
 
+	"github.com/kumahq/kuma/pkg/core/managers/apis/zoneingressinsight"
+
```

Comment:
Nit: I thought our practice was to group all local imports together?

Reply:
Will be fixed by https://github.com/kumahq/kuma/pull/2489, I think we can merge `gci` first and rebase the current PR.